app/[locale]/history/page.tsx

```diff
@@ -1,5 +1,169 @@
+    url: 'https://api.example.com/users/99',
+    latency: '150ms',
+    status: 404,
+    timestamp: '2025-09-09 11:00:12',
+    requestSize: '220B',
+    responseSize: '300B',
+    error: 'Not Found',
+  },
+  {
+    idHistory: '5',
+    method: 'PATCH',
+    full_url: 'http://localhost:3000/PATCH/...',
+    url: 'https://api.example.com/reports',
+    latency: '700ms',
+    status: 500,
+    timestamp: '2025-09-09 11:05:41',
+    requestSize: '400B',
+    responseSize: '0B',
+    error: 'Internal Server Error',
+  },
+  {
+    idHistory: '6',
+    method: 'HEAD',
+    full_url: 'http://localhost:3000/HEAD/...',
+    url: 'https://api.example.com/reports',
+    latency: '700ms',
+    status: 500,
+    timestamp: '2025-09-09 11:05:41',
+    requestSize: '400B',
+    responseSize: '0B',
+    error: 'Internal Server Error',
+  },
+  {
+    idHistory: '7',
+    method: 'OPTIONS',
+    full_url: 'http://localhost:3000/OPTIONS/...',
+    url: 'https://api.example.com/reports',
+    latency: '700ms',
+    status: 500,
+    timestamp: '2025-09-09 11:05:41',
+    requestSize: '400B',
+    responseSize: '0B',
+    error: 'Internal Server Error',
+  },
+];
+
 export default function History() {
+  const t = useTranslations('history');
+
+  if (!mockResponseHistory.length) {
+    return (
+      <div className="flex flex-col justify-center mx-auto">
+        <p className="text-xl font-semibold tracking-tight pb-10">
+          {t('titleNotFound')}:
+        </p>
+        <Button className="mx-auto">
+          <Link href="/rest-client">{t('buttonREST')}</Link>
+        </Button>
+      </div>
+    );
+  }
+
   return (
-    <h1>History & Analytics (private, lazy-loaded, server-side generated)</h1>
+    <div className="flex flex-col justify-center w-full">
+      <h2 className="text-2xl font-semibold tracking-tight first:mt-0 pb-8">
+        {t('title')}
+      </h2>
+      <div className="grid gap-y-0.5">
+        <div className="grid grid-cols-[90px_1fr_90px_90px_0.6fr_100px_105px_1fr] border-t border-b pb-0.5 font-semibold justify-items-center items-center text-center select-none">
+          <p>{t('method')}</p>
+          <p>{t('url')}</p>
+          <p>{t('status')}</p>
+          <p>{t('latency')}</p>
+          <p>{t('timestamp')}</p>
+          <p>{t('requestSize')}</p>
+          <p>{t('responseSize')}</p>
+          <p>{t('error')}</p>
+        </div>
+        {mockResponseHistory.map(
+          ({
+            idHistory,
+            method,
+            full_url,
+            url,
+            status,
+            latency,
+            timestamp,
+            requestSize,
+            responseSize,
+            error,
+          }) => (
+            <Link
+              href={full_url}
+              className="hover:bg-gray-100 grid grid-cols-[90px_1fr_90px_90px_0.6fr_100px_105px_1fr] border-b pb-0.5 justify-items-center"
+              key={idHistory}
+            >
+              <Badge className={badgeVariants[method]}>{method}</Badge>
```

Comment:
```suggestion
<Badge className={`bg-transparent text-sm ${badgeVariants[method]}`}>{method}</Badge>
```

И тогда переменную commonBadgeClasses можно убрать

Reply:
убрала